04week/nasa/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="css/style.css">    
+</head>
+<body>
+<div class="nasa">
+  <img src="img/nasa-background.jpg">
+    <div id="overlay">
+    NASA
+
+    </div>
+</div>
+  <div class="column-layout">
+    <div class="main-column">
+      <h3>OFFICE INFO</h3>
+      <p>
+          Public Communications Office<br>
```

Comment:
I would have used a different tag for each of the lines, that way you could have control over the spacing and padding between the lines